Index: src/main/java/Models/User.java
===================================================================
diff --git a/src/main/java/Models/User.java b/src/main/java/Models/User.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/Models/User.java	
@@ -0,0 +1,4 @@
+package Models;
+
+public class User {
+}
Index: src/main/java/Models/Answer.java
===================================================================
diff --git a/src/main/java/Models/Answer.java b/src/main/java/Models/Answer.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/Models/Answer.java	
@@ -0,0 +1,4 @@
+package Models;
+
+public class Answer {
+}
Index: src/main/java/Models/Question.java
===================================================================
diff --git a/src/main/java/Models/Question.java b/src/main/java/Models/Question.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/Models/Question.java	
@@ -0,0 +1,4 @@
+package Models;
+
+public class Question {
+}
Index: src/main/java/Models/BaseEntity.java
===================================================================
diff --git a/src/main/java/Models/BaseEntity.java b/src/main/java/Models/BaseEntity.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/Models/BaseEntity.java	
@@ -0,0 +1,4 @@
+package Models;
+
+public class BaseEntity {
+}
Index: src/main/java/Models/Quiz.java
===================================================================
diff --git a/src/main/java/Models/Quiz.java b/src/main/java/Models/Quiz.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/Models/Quiz.java	
@@ -0,0 +1,4 @@
+package Models;
+
+public class Quiz {
+}
