Index: src/main/java/com/example/triviaApplication/TriviaApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/TriviaApplication.java b/src/main/java/repository/TriviaApplication.java
rename from src/main/java/com/example/triviaApplication/TriviaApplication.java
rename to src/main/java/repository/TriviaApplication.java
--- a/src/main/java/com/example/triviaApplication/TriviaApplication.java	(revision 9a23778aaf7e1d29d3580d822e8053f652e4d756)
+++ b/src/main/java/repository/TriviaApplication.java	(date 1704674984058)
@@ -1,4 +1,4 @@
-package com.example.triviaApplication;
+package repository;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
Index: src/main/java/repositories/AnswerRepostory.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/repositories/AnswerRepostory.java b/src/main/java/repositories/AnswerRepostory.java
new file mode 100644
--- /dev/null	(date 1704674984062)
+++ b/src/main/java/repositories/AnswerRepostory.java	(date 1704674984062)
@@ -0,0 +1,4 @@
+package repositories;
+
+public class AnswerRepostory {
+}
