Index: src/main/java/com/example/triviaApplication/repositories/QuestionRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/repositories/QuestionRepository.java b/src/main/java/com/example/triviaApplication/repositories/QuestionRepository.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/repositories/QuestionRepository.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.repositories;
+
+public class QuestionRepository {
+}
Index: src/main/java/com/example/triviaApplication/repositories/AnswerRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/repositories/AnswerRepository.java b/src/main/java/com/example/triviaApplication/repositories/AnswerRepository.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/repositories/AnswerRepository.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.repositories;
+
+public class AnswerRepository {
+}
Index: src/main/java/com/example/triviaApplication/models/Quiz.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/models/Quiz.java b/src/main/java/com/example/triviaApplication/models/Quiz.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/models/Quiz.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.models;
+
+public class Quiz {
+}
Index: src/main/java/com/example/triviaApplication/models/Answer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/models/Answer.java b/src/main/java/com/example/triviaApplication/models/Answer.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/models/Answer.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.models;
+
+public class Answer {
+}
Index: src/main/java/com/example/triviaApplication/models/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/models/User.java b/src/main/java/com/example/triviaApplication/models/User.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/models/User.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.models;
+
+public class User {
+}
Index: src/main/java/com/example/triviaApplication/models/BaseEntity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/models/BaseEntity.java b/src/main/java/com/example/triviaApplication/models/BaseEntity.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/models/BaseEntity.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.models;
+
+public class BaseEntity {
+}
Index: src/main/java/com/example/triviaApplication/models/Question.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/models/Question.java b/src/main/java/com/example/triviaApplication/models/Question.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/models/Question.java	
@@ -0,0 +1,4 @@
+package com.example.triviaApplication.models;
+
+public class Question {
+}
Index: src/main/resources/templates/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/index.html b/src/main/resources/templates/index.html
new file mode 100644
--- /dev/null	
+++ b/src/main/resources/templates/index.html	
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
\ No newline at end of file
Index: src/main/java/com/example/triviaApplication/controllers/HomeController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/controllers/HomeController.java b/src/main/java/com/example/triviaApplication/controllers/HomeController.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/controllers/HomeController.java	
@@ -0,0 +1,14 @@
+package com.example.triviaApplication.controllers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.RequestMapping;
+@Controller
+public class HomeController {
+    @GetMapping("/")
+    public String home() {
+        return "index"; // Return the name of the HTML file (index.html)
+    }
+}
Index: src/main/java/com/example/triviaApplication/controllers/UserQuestionController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/controllers/UserQuestionController.java b/src/main/java/com/example/triviaApplication/controllers/UserQuestionController.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/example/triviaApplication/controllers/UserQuestionController.java	
@@ -0,0 +1,10 @@
+package com.example.triviaApplication.controllers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.RequestMapping;
+@Controller
+public class UserQuestionController {
+}
