Index: src/main/java/Repository/AnswerRepository.java
===================================================================
diff --git a/src/main/java/Repository/AnswerRepository.java b/src/main/java/Repository/AnswerRepository.java
deleted file mode 100644
--- a/src/main/java/Repository/AnswerRepository.java	
+++ /dev/null	
@@ -1,2 +0,0 @@
-package Repository;public interface ANswerRepository {
-}
Index: src/main/java/repositories/AnswerRepostory.java
===================================================================
diff --git a/src/main/java/repositories/AnswerRepostory.java b/src/main/java/repositories/AnswerRepostory.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/repositories/AnswerRepostory.java	
@@ -0,0 +1,4 @@
+package repositories;
+
+public class AnswerRepostory {
+}
Index: src/main/java/com/example/triviaApplication/TriviaApplication.java
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/TriviaApplication.java b/src/main/java/repository/TriviaApplication.java
rename from src/main/java/com/example/triviaApplication/TriviaApplication.java
rename to src/main/java/repository/TriviaApplication.java
--- a/src/main/java/com/example/triviaApplication/TriviaApplication.java	
+++ b/src/main/java/repository/TriviaApplication.java	
@@ -1,4 +1,4 @@
-package com.example.triviaApplication;
+package repository;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
