Index: src/main/java/com/example/triviaApplication/TriviaApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/triviaApplication/TriviaApplication.java b/src/main/java/Repository/TriviaApplication.java
rename from src/main/java/com/example/triviaApplication/TriviaApplication.java
rename to src/main/java/Repository/TriviaApplication.java
--- a/src/main/java/com/example/triviaApplication/TriviaApplication.java	(revision da4382da6897bc7f389240a55d6e12690259b5c9)
+++ b/src/main/java/Repository/TriviaApplication.java	(date 1704674603605)
@@ -1,4 +1,4 @@
-package com.example.triviaApplication;
+package Repository;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
